acceptance/bundle/exec/databricks-cli/profile-is-passed/from_bundle/output.txt

```diff
@@ -0,0 +1,6 @@
+
+>>> [CLI] bundle exec -- [CLI] current-user me
```

Comment:
We should actually do a further review of the name of this command

Reply:
Moved to bundle run